packages/caravan-fees/src/types.ts

```diff
@@ -442,3 +433,36 @@ export interface CPFPOptions {
    */
   strict?: boolean;
 }
+
+/**
+ * Comprehensive object containing all supported Bitcoin script types.
+ * This includes multisig address types from Caravan and additional types.
+ *
+ * @readonly
+ * @enum {string}
+ */
+export const SCRIPT_TYPES = {
+  /** Pay to Public Key Hash */
+  P2PKH: "P2PKH",
+  /** Pay to Witness Public Key Hash (Native SegWit) */
+  P2WPKH: "P2WPKH",
+  /** Pay to Script Hash wrapping a Pay to Witness Public Key Hash (Nested SegWit) */
+  P2SH_P2WPKH: "P2SH_P2WPKH",
+  /** Unknown or unsupported script type */
+  UNKNOWN: "UNKNOWN",
+  /** Pay to Script Hash */
+  P2SH: MULTISIG_ADDRESS_TYPES.P2SH as "P2SH",
```

Comment:
why was the `as` needed here?

Reply:
I was facing some type-error so tried type assertion to correct that ... but it seems that was due to a wrong type I was using ... corrected this [now](https://github.com/caravan-bitcoin/caravan/pull/114/commits/73dd32398d39f745d2ab5083a791c1a14a9827cc#diff-593f8d602f801c56261ed7a8dc941947102f6a7b26c483c5ee8cf9c293fffb33R418)